src/Compilers/CSharp/Test/Symbol/Symbols/MissingSpecialMember.cs

```diff
@@ -570,6 +570,9 @@ public void AllWellKnownTypes()
                     case WellKnownType.ExtSentinel:
                         // Not a real type
                         continue;
+                    case WellKnownType.System_Runtime_CompilerServices_ITuple:
```

Comment:
Actually, ITuple has shipped ([more details on where](https://github.com/dotnet/roslyn/issues/13177)). And it may be available in ValueTupleRef which is referenced above (I don't remember exactly what that ref was based off of).